content/chainguard/chainguard-images/shared-responsibility-model/index.md

```diff
@@ -0,0 +1,167 @@
+weight: 575
+toc: true
+---
+
+Chainguard's Shared Responsibility Model is a comprehensive framework that delineates security responsibilities for hardened container images among Chainguard, upstream open-source projects, and customers. This model provides a clear understanding of who is responsible for what, ensuring a secure and compliant container environment. It addresses two primary categories: App Images (used as-is) and Base Images (used in multistage builds).
+
+This reference guide outlines the Shared Responsibility Model, and includes a helpful diagram outlining the relevant responsiblities. It is intended to serve as a resource for customers who want to better understand their security responsibilities (as well as Chainguard's responsibilities) in regards to Chainguard Images. 
+
+
+## Hardened Container Image Shared Responsibility Model (Diagram)
+
+![Diagram representing Chainguard's shared responsibility model. On the left is a pyramid structure labeled "App Images" with 4 tiers: Apps, LangLibs (go mod, pom, …), Toolchains (go, java, php, …), and System (glibc, openssl). To the left of the pyramid there are two brackets showing that CVE management is performed by the Upstream Open Source project at the Apps level and by Chainguard at the Toolchains and System level. To the right of the pyramid are two smaller brackets showing that if bumping a dependency is breaking, the CVE management falls to the Upstream project; if it's successful, then CVE management falls to Chainguard. On the right is another pyramid labeled "Base images" with 3 tiers: Custom code, Toolchains (go, java, php, …), and System (glibc, openssl, …). To the left of this pyramid are two brackets, showing that CVE management is the responsibility of the customer a the Custom code level and it's the responsibility of Chainguard at the Toolchains and System levels.](shared-responsibility-model.png)
+
+The next few sections dig deeper into the details about App Images, Base Images, and the responsibilities of Chainguard and its customers for both.
+
+
+## App Images
+Chainguard App Images are specialized container images designed to be used "as-is" in production environments.
+
+### Key Characteristics
+1. Minimal Attack Surface: Built using a [distroless](/chainguard/chainguard-images/getting-started-distroless/) approach, removing unnecessary components to reduce potential vulnerabilities.
+2. Up-to-date Components: Continuously updated with the latest security patches and version updates from upstream projects.
+3. Low to Zero CVEs: Rigorously scanned and maintained to minimize known vulnerabilities.
+4. Layered Security Responsibility:
+    * Top layers (Apps and Lang Libs) are managed by upstream open source projects
+    * Lower layers (Toolchains and System) are secured and maintained by Chainguard
+5. Compliance with CHP Levels: Adhere to Chainguard's Container Hardening Policy, typically at higher levels (CHP3-CHP5) for enhanced security.
```

Comment:
I thought CHPs this was just a proposal! Mention that they are explained below, maybe just link?